### PR TITLE
#55 register view

### PR DESCRIPTION
**Pull request by LukaszAmbrozewski**

### CODE REVIEW

File: src/components/ChangePasswordBox/ChangePasswordBox.tsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { SubmitHandler, useForm } from 'react-hook-form';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { userSchema } from '../../Validations/UserValidation';
 import { TextField } from '@mui/material';
-import { styled } from '@mui/system';
-import './ChangePasswordBox.css';
-import { SmallBtn } from '../../common/SmallBtn/SmallBtn';
+import {styled} from '@mui/system';
+import '../commonStyles.css';
+import {SmallBtn} from '../../common/SmallBtn/SmallBtn';
```
Comment (patrycja-dev41171) [inline src/components/ChangePasswordBox/ChangePasswordBox.tsx:8]: Good use of new packages.

File: src/components/ChangePasswordBox/ChangePasswordBox.tsx [hunk 2]
```diff
@@ -70,45 +70,46 @@ export const ChangePasswordBox = () => {
...
+            <StyledTextField
+                variant="outlined"
+                type="password"
+                error={!!errors.confirmPassword}
+                label="Potwierdź hasło"
+                helperText={errors.confirmPassword && 'Podane hasła nie są takie same!'}
+                {...register('confirmPassword')}
+                disabled={isSuccessfullySubmitted}
+            />
+          </div>
```
Comment (patrycja-dev41171) [inline src/components/ChangePasswordBox/ChangePasswordBox.tsx:105]: Good structure of component.

File: src/components/RegisterBox/RegisterBox.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+          registerToken,
+        }),
+      });
+      const result = await res.json();
+      setFeedback(result.message);
+      setIsSuccessfullySubmitted(true);
+    } catch (err) {
+      console.log(err);
+    }
+  };
```
Comment (patrycja-dev41171) [inline src/components/RegisterBox/RegisterBox.tsx:71]: Support for sending data to BE. Great!

**Review by patrycja-dev41171 — COMMENTED** [review]
Great code!

File: src/pages/RegisterView/RegisterView.tsx [hunk 4]
```diff
@@ -1,11 +1,6 @@
...
-  return (
-    <>
-      <h1>Register View</h1>
-      <p>Strona rejestracji student/hr z linku w email. Formularz (hasło/hasło)</p>
-      <p>Dostęp: Niezalogowany użytkownik który ma link.</p>
-    </>
-  );
-};
+  return <RegisterBox/>;
+};
```
Comment (MiroslawChmielecki) [inline src/pages/RegisterView/RegisterView.tsx:6]: This is ok, pages views should includes only ready components

File: src/components/commonStyles.css [hunk 5]
```diff
@@ -1,4 +1,4 @@
-.changePass-view {
```
Comment (MiroslawChmielecki) [inline src/components/commonStyles.css:1]: In the components folder we should only have separate components in directories and this css file doesn't fit here

**Review by MiroslawChmielecki — APPROVED** [review]

**Pull request merged**